### ISSUE TITLE
Add ability to focus on Message component

### ISSUE THREAD

**Issue by phil9690** [Sage/carbon#5924]
### Desired behaviour

We need the ability to be able to set a focus state on a Message component.

Context: To follow a11y guidelines in situations where an action is performed by the user and the action has finished, if a message component is then rendered to inform the user of the completed action, the message should be focused on. Anyone using a screen reader in this case would get confirmation that the action has finished.

### Current behaviour

Message component is not focusable.

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
FE-5725
